src/help.h

```diff
@@ -1180,7 +1180,7 @@ struct commandHelp {
     "3.2.1" },
     { "TTL",
     "key",
-    "Get the time to live for a key",
+    "Get the time to live for a key in seconds",
```

Comment:
This is actually mastered in a separate file, https://github.com/redis/redis-doc/blob/7282fadc4b1e1a2c738b0790c292b288d2fe6b68/commands.json#L3982, and we update it periodically. Please open the PR against the redis-doc repository.